themes/default/content/learn/building-with-pulumi/secrets/index.md

```diff
@@ -289,4 +288,4 @@ programmatically, review the
 From here, we're moving on to the last part of the Pulumi in Practice pathway:
 testing. Onward!
 
-<!-- [^1]: [state]({{< relref "/docs/intro/concepts/state" >}}) -->
\ No newline at end of file
+<!-- [^1]: [state]({{< relref "/docs/intro/concepts/state" >}}) -->
```

Comment:
Was this branch from an older version of the pathways? I could have sworn I took these out so we had time to investigate footnotes in terms of UX, so I was surprised to see them pop up here.

Reply:
This is branched from latest master, and I see this line there (and in a few other places, looks like), so maybe just snuck past ya. Want me to remove them? Or maybe in a separate PR?

Reply:
Nah, we can leave them for now, I think. I must have missed some :)